src/datumaro/plugins/specs.json

```diff
@@ -1,13 +1,4 @@
 [
-  {
-    "import_path": "datumaro.plugins.accuracy_checker_plugin.ac_launcher.AcLauncher",
-    "plugin_name": "ac",
-    "plugin_type": "Launcher",
-    "extra_deps": [
-      "tensorflow",
-      "openvino.tools.accuracy_checker"
-    ]
-  },
   {
```

Comment:
Is it related to kitti 3d? Why is this removed?

Reply:
This is not related to kitti3d, I have fixed this in [a9727ff](https://github.com/openvinotoolkit/datumaro/pull/1619/commits/a9727ff8b6155ceab950d0b0bedea0ab400a0d0c). Thanks for bringing this to my attention.